### PR TITLE
Updating light blinking docs for SES 7

### LINKED ISSUE #535: [doc] 14.10  How to Locate Physical Disks Using LED Lights
### Type of report
   - [x] bug
   - [ ] feature request

### Source Document
14.10  How to Locate Physical Disks Using LED Lights:

https://susedoc.github.io/doc-ses/master/html/ses-troubleshooting/storage-tips.html#bp-flash-led-lights

### Summary
Has updated to: https://github.com/ceph/ceph/blob/master/src/pybind/mgr/cephadm/module.py#L1810-L1818

### PR DESCRIPTION
**Pull request by asettle**
Closes #535

### CODE REVIEW

File: xml/bp_troubleshooting_tips.xml [hunk 1]
```diff
@@ -548,242 +548,21 @@ section. A.S -->
   <title>How to Locate Physical Disks Using LED Lights</title>
 
   <para>
-   This section describes using <systemitem>libstoragemgmt</systemitem> and/or
-   third party tools to adjust the LED lights on physical disks. This
-   capability may not be available for all hardware platforms.
+    &ceph; tracks which hardware storage devices (HDDs, SSDs) are consumed
+    by which daemons, and collects health metrics about those devices in
```
Comment (tbazant) [inline xml/bp_troubleshooting_tips.xml:552]:
```suggestion
    &ceph; tracks which daemons manage which hardware storage devices (HDDs, SSDs), and collects health metrics about those devices in
```

File: xml/bp_troubleshooting_tips.xml [hunk 2]
```diff
@@ -548,242 +548,21 @@ section. A.S -->
...
-    [[ "$FOUND" = "1" ]] &amp;&amp; break
-  fi
-done</screen>
-
+</screen>&prompt.cephuser;ceph device light --enable=on --devid=string --light_type=ident --force</screen>
   <para>
-   Sample script for flashing LED lights on LSI hardware using the
-   <systemitem>storcli</systemitem> utilities:
+    The <literal>devid</literal> parameter is the device identification. You
+    can obtain this information using the following command:
```
Comment (tbazant) [inline xml/bp_troubleshooting_tips.xml:562]:
```suggestion
    The <option>DEVID</option> parameter is the device identification. You
    can obtain it by running the following command:
```

**Review by tbazant — CHANGES_REQUESTED** [review]
just nitpicks :-)

**Review by tbazant — APPROVED** [review]
lgtm

**Review by votdev — APPROVED** [review]

**Pull request merged**